akka-projection-core/src/main/scala/akka/projection/internal/OffsetSerialization.scala

```diff
@@ -75,11 +75,14 @@ import akka.projection.StringKey
       case seq: query.Sequence      => SingleOffset(id, SequenceManifest, seq.value.toString, mergeable)
       case tbu: query.TimeBasedUUID => SingleOffset(id, TimeBasedUUIDManifest, tbu.value.toString, mergeable)
       case mrg: MergeableOffset[_, _] =>
-        MultipleOffsets(mrg.entries.map {
+        val list = mrg.entries.map {
           case (key: MergeableKey, innerOffset) =>
             toStorageRepresentation(ProjectionId(id.name, key.surrogateKey), innerOffset, mergeable = true)
               .asInstanceOf[SingleOffset]
-        }.toSeq)
+        }.toList
+
+        MultipleOffsets(list)
```

Comment:
map.toSeq is mutable in 2.12. 

I pull it out from the constructor to reduce some letter-soup. Hard to parse for the brackets when all together.